micro-ui/web/micro-ui-internals/packages/modules/core/src/components/TopBarSideBar/SideBar/EmployeeSideBar.js

```
@@ -94,61 +70,119 @@ const EmployeeSideBar = () => {
+        label: key,
+        icon: { icon: iconKey, width: "1.5rem", height: "1.5rem" },
+        children: children,
+      };
+    };
+    return Object.keys(data).map((key) => transformItem(key, data[key]));
+  }
+
+  const sortDataByOrderNumber = (data) => {
+    // Sort the current level of data by orderNumber, handling cases where orderNumber might be missing
+    data.sort((a, b) => {
+      const aOrder = a.orderNumber !== undefined ? a.orderNumber : Infinity; // Use Infinity if orderNumber is missing
```

Biome

[error] Use Number.Infinity instead of the equivalent global.

ES2015 moved some globals into the Number namespace for consistency.
Safe fix: Use Number.Infinity instead.


(lint/style/useNumberNamespace)